frontend/src/components/SummaryView/index.jsx

```diff
@@ -137,7 +179,12 @@ function SummaryView({
     }
 
     svg.append('g').call(yAxis);
-    svg.append('g').call(xAxis);
+    svg.append('g').call(xAxis)
+      .selectAll("text")  
+      .style("text-anchor", "end")
+      .attr("dx", "-.8em")
+      .attr("dy", ".15em")
+      .attr("transform", "rotate(-70)");
```

Comment:
We should also show the x-axis label type. Like if its genre/artist name etc.

Reply:
Also, the font size for the label ticks is too small to be legible. Maybe we increase it to at least 12px?